File: app/src/main/java/com/privat/contacts/presentation/favorite/FavoriteView.java
package com.privat.contacts.presentation.favorite;

import com.privat.contacts.presentation.base.MvpView;
import com.privat.contacts.presentation.users.model.UserItemUi;

import java.util.List;

public interface FavoriteView extends MvpView {
    void showFavoriteList(List<UserItemUi> userItemUis);
}

File: app/src/main/java/com/privat/contacts/presentation/favorite/FavoriteView.kt
package com.privat.contacts.presentation.favorite

import com.privat.contacts.base.presentation.MvpView
import com.privat.contacts.presentation.users.model.UserItemUi

interface FavoriteView : MvpView {
    fun showFavoriteList(userItemUis: List<UserItemUi>)
}
